### PR TITLE
Add awesome share links

### PR DESCRIPTION
**Pull request by vinkla**
https://github.com/vinkla/awesome-share-links

### CODE REVIEW

**Comment by aleksandar-todorovic** [comment]
It needs a license (which is currently tough to implement since there are 59 contributors) and clear contributing guidelines. As for the content, this is more then useful and I definitely think it should be a part of this list.

Pinging @PharkMillups to see if he's interested in adding those.

**Comment by vinkla** [comment]
It has both license and contribution guidelines?

**Comment by sindresorhus** [comment]
MIT is not a good license for normal text content. Read this: https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license

**Comment by vinkla** [comment]
Okay, I'll change.

**Comment by vinkla** [comment]
@sindresorhus Okay, it has been changed.

File: readme.md [hunk 1]
```diff
@@ -305,6 +305,7 @@ If you want to create your own list, please read the [create list instructions](
 - [Research Tools](https://github.com/emptymalei/awesome-research)
 - [Niche Job Boards](https://github.com/wfhio/awesome-job-boards)
 - [Data Visualization](https://github.com/fasouto/awesome-dataviz)
+- [Share Links](https://github.com/vinkla/awesome-share-links)
```
Comment (sindresorhus) [inline readme.md]: `Share Links` => `Social Media Share Links` to make it clearer.

Comment (vinkla) [inline readme.md]: Done.

Comment (vinkla) [inline readme.md]: @sindresorhus Done.

**Comment by sindresorhus** [comment]
Would be awesome if you would add the awesome badge somewhere in your list ;)

https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge

**Comment by vinkla** [comment]
Thanks!

**Pull request merged**